### PR TITLE
README updates for v0.2

### PR DESCRIPTION
**Pull request by danforbes**
Rendered version can be seen here https://github.com/danforbes/llama-rs/blob/dfo/docs/readme/README.md

### CODE REVIEW

File: README.md
Comment (philpax) [inline README.md:65]: I think this isn't transitive, so try `ggml-sys` for the maximum impact. We should test this.

File: README.md
Comment (philpax) [inline README.md:134]: Should this be gptj?

File: README.md
Comment (philpax) [inline README.md:148]: LLaMA model only - we don't have scripts for the other models

File: README.md
Comment (philpax) [inline README.md:177]: more more -> more

File: README.md
Comment (philpax) [inline README.md:203]: Flake -> nixOS flake

File: README.md
Comment (philpax) [inline README.md:4]: its -> it's

File: README.md
Comment (philpax) [inline README.md:71]: command line -> command-line

We should also mention downloading the CLI from the GitHub releases.

File: README.md
Comment (philpax) [inline README.md:38]: Can we list some interesting examples of these? BLOOM has BLOOMZ, GPT-J has GPT4All, etc

**Review by philpax — CHANGES_REQUESTED** [review]

File: README.md
Comment (danforbes) [reply README.md:134]: Nope

File: README.md
Comment (danforbes) [reply README.md:38]: I don't understand this request.

Comment (philpax) [reply README.md:40]: So the README has

> Currently, the following models are supported:
> 
>     [BLOOM](https://huggingface.co/docs/transformers/model_doc/bloom)
>     [GPT-2](https://huggingface.co/docs/transformers/model_doc/gpt2)
>     [GPT-J](https://huggingface.co/docs/transformers/model_doc/gptj)
>     [GPT-NeoX](https://huggingface.co/docs/transformers/model_doc/gpt_neox): GPT-NeoX, StableLM, RedPajama, Dolly v2
>     [LLaMA](https://huggingface.co/docs/transformers/model_doc/llama): LLaMA, Alpaca, Vicuna, Koala, GPT4All, GPT4-X, Wizard
>     [MPT](https://www.mosaicml.com/blog/mpt-7b)

where GPT-NeoX and LLaMA have some examples of models that use those architectures. It would be nice to have some examples for BLOOM, GPT-2, GPT-J and MPT too (even if it's just "MPT" for the last one), just so that it doesn't look weirdly imbalanced.

They could also be links to the descriptions of the models, but I leave that up to you.

File: README.md
Comment (philpax) [inline README.md:84]: Add a section here to download the CLI from Releases, with the caveat that it might not be the most up-to-date

**Review by philpax — APPROVED** [review]
Pretty much good to go, just those last two things. Once they're sorted I'll merge

**Pull request merged**
